exercises/nth-prime/src/Prime.hs

```diff
@@ -1,3 +1,4 @@
 module Prime (nth) where
 
-nth = error "You need to implement this function."
+nth :: Int -> Maybe Int
```

Comment:
The input can be any `Integral` type, but the test suite demands the output to be `Maybe Integer`.

This is possiblywhat makes this PR fail in Travis.